vscode-dotnet-runtime-library/distro-data/distro-support.json

```
@@ -1,5 +1,401 @@
 {
     "$schema": "../../Documentation/json-schema/distro-support-schema.json",
+    "Debian": {
```

```suggestion
    "Debian GNU/Linux": {
```

My Debian machine seems to show this name in etc/os-release. Does yours only show Debian? This doesn't work unless this matches that title. Do you know if there's a case when this would/wouldn't have {space}GNU/Linux?

---

you are right , i though it should be the Id of distro but it seems that it is  Name,
i don't know any other cases for name but i guess if we change  distroName to distroId we will not have  any inconsistency

---

I think this should be good on my research. Itd be nice to have official confirmation from an expert but also I dont think it matters that much.